.editorconfig

```diff
@@ -7,5 +7,17 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[SourceX/miniwin/ddraw.*]

```

Comment:
in stead of canonicalizing this the files should just be corrected, I'm happy with you doing this in a single commit just for that, or just create an issue for it.

Reply:
Issue created https://github.com/diasurgical/devilutionX/issues/262

We should probably wait until the big PRs are merged before fixing this though because resolving conflicts and rebasing is hell when line endings change.

Reply:
I'm happy to resolve stuff like that but sure no rush on my end 😊